README.md

```diff
@@ -11,7 +19,38 @@ import numpy as np
 x_sample = np.random.normal(size = (500, 10))
 y_sample = np.random.normal(size = (400, 10))
 
+# To get pvalues from PQMass
 pvalues = pqm_pvalue(x_sample, y_sample, num_refs = 100, bootstrap = 50)
-
 print(np.mean(pvalues), np.std(pvalues))
+
+# To get chi2 from PQMass
+chi2_stat = pqm_chi2(x_sample, y_sample, num_refs = 100, bootstrap = 50)
+print(np.mean(chi2_stat), np.std(chi2_stat))
+```
+
+If your two samples are drawn from the same distribution then the pvalue should
+be drawn from the random uniform(0,1) distribution. This means that if you get a
+very small value (i.e. 1e-6) then you have failed the null hypothesis test and
+the two samples are not drawn from the same distribution.
+
+For the chi2 metric, given your two sets of samples, if they come from the same
+distribution, the histogram of your chi² values should follow the chi² distribution. 
+The peak of this distribution will be at DoF -  and the standard deviation will 
+be √(2 * DoF). If your histogram shifts to the right of the expected chi² distribution, 
+it suggests that the samples are out of distribution. Conversely, if the histogram shifts 
+to the left, it indicates potential duplication or memorization (particularly relevant 
+for generative models).
+
+Note that if you have low number of samples, we highly recommend using pvalue as the 
```

Comment:
I don't think the chi^2 will be biased. Although I think if they have a low number of samples they should make sure to use a lot of bootstrap iterations, then they will get a sense of the uncertainty on the chi^2 or pvalue